app/lib/smaxtec_api.rb

```diff
@@ -32,8 +106,11 @@ def update_sensor_readings
 
   def smaxtec_sensor_readings(sensor)
     metric = PROPERTY_MAPPING[sensor.property]
+    event = EVENT_MAPPING[sensor.property]
     unless metric
-      puts "Sensor #{sensor.name} has unrecognized property: #{sensor.property}"
+      if !event
```

Comment:
Because in this method we only want to fetch the sensor readings and not the events. Because we iterate over all the sensors (including the event sensors), we need to make sure we don't fetch any event sensor, because the events are handled in the smaxtec_events method.
For debugging reasons (in the log), we want to know if there is a sensor which has an unrecognized property, this is why we iterate over all available sensors. But we need to make sure we don't show the error message, if this is an event sensor, because in this case it isn't an unrecognized property, but just an event which don't care about in this method.